tests/trainer/test_trainer_eval.py

```diff
@@ -52,6 +52,30 @@ def test_eval_call():
     assert trainer.state.eval_metrics['eval']['MulticlassAccuracy'].compute() != 0.0
 
 
+@world_size(1, 2)
```

Comment:
could you add an FSDP parametrization to this test, just to be safe?

Reply:
Will combine with ICL

Reply:
Discussed offline, fine to skip